keel-core/src/main/kotlin/com/netflix/spinnaker/keel/events/PersistentEventListener.kt

```diff
@@ -5,9 +5,9 @@ import org.springframework.context.event.EventListener
 import org.springframework.stereotype.Component
 
 @Component
-class ApplicationEventListener(private val resourceRepository: ResourceRepository) {
-    @EventListener(ApplicationEvent::class)
-    fun onResourceEvent(event: ApplicationEvent) {
+class PersistentEventListener(private val resourceRepository: ResourceRepository) {
```

Comment:
There was redundant code to persist different sub-classes of `PersistentEvent`. This simplifies that a bit.